tools/clang/test/CodeGenSPIRV/empty-void-main.hlsl2spv

```diff
@@ -6,8 +6,7 @@ void main()
 
 
 // TODO:
-// OpEntryPoint Fragment %main "main"
-// OpExecutionMode %main OriginUpperLeft
+// Implement logic to determine ExecutionMode (currently using OriginUpperLeft by default)
```

Comment:
This shader is essentially done; even if we implemented the logic for determining execution mode, the execution mode here is still `OriginUpperLeft` I think. So there is no need to still put TODO here.